### ISSUE TITLE
Warning on docker version during preflights is wrong

### ISSUE THREAD

**Issue by chuckha** [kubernetes/kubeadm#1131]
## Is this a BUG REPORT or FEATURE REQUEST?

BUG REPORT

## Versions

**kubeadm version** (use `kubeadm version`): v1.12.0-rc.1

docker version: 17.03.3-ce

## What happened?
When running kubeadm init the preflight check says:

```
        [WARNING SystemVerification]: docker version is greater than the most recently validated version. Docker version: 17.03.3-ce. Max validated version: 18.06
```

## What you expected to happen?
I expected it to to not warn or to tell me it's *less than* the most recently validated version, not greater than.

## How to reproduce it (as minimally and precisely as possible)?
* Install docker 17.03
* set up a kubelet with v1.12.0-rc.1 or later
* run kubeadm init with v1.12.0-rc.1 or later

## Anything else we need to know?
This is a non urgent bug and it's a good first issue so I'm assigning it to @carlisia.

**Comment by timothysc** [comment]
@neolit123 does this even exist now with your last PR fix for 18.06?

**Comment by neolit123** [comment]
@timothysc 
yep, i modified the message as it was incorrect.

right now it indicates that the tested version is not in the list of versions that are known to work:
https://github.com/kubernetes/kubernetes/blob/master/cmd/kubeadm/app/util/system/docker_validator.go#L81-L83

/close

**Comment by neolit123** [comment]
@carlisia, @chuckha sorry we had to fix that as it was urgent for 1.12. :)
